scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -1,11 +1,16 @@
 :_content-type: ASSEMBLY
 [id="enabling-workload-partitioning"]
 = Workload partitioning
+:FeatureName: Workload partitioning
+include::snippets/technology-preview.adoc[]
+:!FeatureName: Workload partitioning
 include::_attributes/common-attributes.adoc[]
 :context: enabling-workload-partitioning
 
 toc::[]
 
+:FeatureName: Workload partitioning
```

Comment:
This should not have been added. This just resets the `FeatureName` variable and could lead to issues in other parts of the doc.

Reply:
Fixed in https://github.com/openshift/openshift-docs/pull/59932